ims_python_helper/__init__.py

```diff
@@ -316,17 +404,35 @@ def image_upload_artifacts(
         """
 
         # Stub out the return value of this method
+        if rootfs:
+            rootfs = rootfs[0]
+        if kernel:
+            kernel = kernel[0]
+        if initrd:
+            initrd = initrd[0]
+
```

Comment:
Keep the comment with the code it's commenting on.

```suggestion
        if rootfs:
            rootfs = rootfs[0]
        if kernel:
            kernel = kernel[0]
        if initrd:
            initrd = initrd[0]

        # Stub out the return value of this method
```